dwave/system/samplers/dwave_sampler.py

```diff
@@ -21,8 +21,9 @@
 """
 from __future__ import division
 
-import functools
 import time
+import functools
```

Comment:
I do alphabetical, you do by size :laughing:

Reply:
Not true at all.

![image](https://user-images.githubusercontent.com/447371/93938980-f31e2600-fcde-11ea-8bd5-1f805ef4ace3.png)